server/src/weaverbird/backends/pypika_translator/translators/googlebigquery.py

```diff
@@ -66,13 +66,9 @@ def _get_single_condition_criterion(
         if isinstance(condition, DateBoundCondition):
             match condition.operator:
                 case "from":
-                    return functions.Cast(column_field, "datetime") >= ParseDatetime(
-                        "%FT%T", condition.value
-                    )
+                    return functions.Cast(column_field, "timestamp") >= condition.value
                 case "until":
-                    return functions.Cast(column_field, "datetime") <= ParseDatetime(
-                        "%FT%T", condition.value
-                    )
+                    return functions.Cast(column_field, "timestamp") <= condition.value
```

Comment:
Oh sweet if this works, I don't like the `%FT%T` format string because I don't understand it :sweat_smile: